styles.css

```diff
@@ -62,6 +269,16 @@ body {
     margin-top: 4vh;
   }
 
+  .heading {
+    color: #172b4d;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
```

Comment:
[ OPTIONAL ] for responsiveness always use relative units like rem for font size to make your text scale .